test/check-application

```diff
@@ -302,6 +304,25 @@ WantedBy=multi-user.target default.target
         self.execute(auth, f"{systemctl_cmd} daemon-reload; {systemctl_cmd} start {name}.service")
         self.execute(auth, f"until {systemctl_cmd} is-active {name}.service; do sleep 1; done")
 
+    def createQuadletPod(self, name: str, *, auth: bool = False) -> None:
+        quadlet = f"""
+[Pod]
+PodName={name}
+
+[Install]
+WantedBy=multi-user.target default.target
```

Comment:
This is an odd WantedBy (it should have one or the other), but we also don't need an Install section to test this at all -- we don't do any reboot/autostart tests here.

Reply:
FTR, this was just thinking aloud -- if you want to keep it, do, it's not important either way.